From: Lennart Poettering <lennart@poettering.net>
Date: Thu, 26 Sep 2013 18:31:37 +0000
Subject: logind: if a user is sitting in front of the computer and can
 shutdown the machine anyway he should also be able to reboot it

(cherry picked from commit 299404a19f26aa4f203042d8285ee0b7afa5bf40)
---
 src/login/org.freedesktop.login1.policy.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/login/org.freedesktop.login1.policy.in b/src/login/org.freedesktop.login1.policy.in
index b96d32d..df12398 100644
--- a/src/login/org.freedesktop.login1.policy.in
+++ b/src/login/org.freedesktop.login1.policy.in
@@ -201,7 +201,7 @@
                 <defaults>
                         <allow_any>auth_admin_keep</allow_any>
                         <allow_inactive>auth_admin_keep</allow_inactive>
-                        <allow_active>auth_admin_keep</allow_active>
+                        <allow_active>yes</allow_active>
                 </defaults>
                 <annotate key="org.freedesktop.policykit.imply">org.freedesktop.login1.reboot</annotate>
         </action>
